mmv1/templates/terraform/examples/firestore_field_basic.tf.erb

```diff
@@ -0,0 +1,17 @@
+resource "google_firestore_field" "<%= ctx[:primary_resource_id] %>" {
+  project = "<%= ctx[:test_env_vars]['project_id'] %>"
+  collection = "chatrooms_%{random_suffix}"
+  field = "basic"
+
+  index_config {
```

Comment:
👍🏻 on all of the examples included. If possible, could we also add one that explicitly shows how to create an SFI exclusion?

Reply:
Note these are all technically exclusions (they do not inherit from the ancestor configuration). The timestamp example covers the case where a user wants to completely disable their single field indexes. I added a comment there (and updated the description in the yaml) to make this more obvious.